exporter/prometheusexporter/config.go

```diff
@@ -44,10 +45,19 @@ type Config struct {
 
 	// ResourceToTelemetrySettings defines configuration for converting resource attributes to metric labels.
 	ResourceToTelemetrySettings resourcetotelemetry.Settings `mapstructure:"resource_to_telemetry_conversion"`
+
+	// sanitizeLabel specifies whether to sanitize labels that start with _
+	sanitizeLabel bool
```

Comment:
This description feels backward.  If this value is false then the labels will be sanitized.  If it is true, they will not.

Reply:
updated.